src/space_getsid.h

```diff
@@ -79,4 +78,90 @@ space_getsid_and_swap_cells(const struct space *s, struct cell **ci,
   return sid;
 }
 
+__attribute__((always_inline, nonnull)) INLINE static int // A. Nasar Same as usual but only used to pack GPU cells
+space_getsid_GPU(const struct space *s, struct cell **ci,
+             struct cell **cj, double *shift_x, double *shift_y,
+			 double *shift_z) {
```

Comment:
Why do we need a separate function?

Reply:
I was playing around with different ways of getting the packing to go faster using float3 variables but that didn't work